bip78/Cargo.toml

```diff
@@ -18,12 +18,13 @@ sender = []
 receiver = ["rand"]
 
 [dependencies]
-bitcoin = "0.27.1"
+bitcoin = { version = "0.28.1", features = ["base64"] }
```

Comment:
FYI this is not really that great: https://github.com/Kixunil/loptos/issues/1#issuecomment-888896092